src/utilities/uploadImage.ts

```diff
@@ -33,6 +34,7 @@ export const uploadImage = async (
         ),
       )
       .on("close", resolve)
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why was this added? How can you rescue the code to eliminate the explicit any?

Reply:
I wasn't able to commit without removing this error locally.

Reply:
I have added they type to paramater `error` now